README.md

```diff
@@ -76,6 +76,25 @@ the relevant html reports, for instance:
     - file:///foo/project/build/reports/pmd/main4.html
 ```
 
+It's possible to specify a custom renderer for the report urls in the logs via the `logs` extension. This can be useful in CI
+ environments, where the local paths are not reachable directly. For instance the snippet below will replace the base url with
+ one of your choice:
+```gradle
+staticAnalysis {
+    ...
+    logs {
+        reportBaseUrl "http://ci.mycompany.com/job/myproject/ws/app/build/reports"
```

Comment:
The example follows the implementation of `reportBaseUrl` as convenience method built on top of the more generic `reportUrlRenderer` facility, so it's correct. The emphasis is put on the reports location rather than the project location as you are suggesting.
I'm happy to remove this utility and/or change the sample, but having `reportBaseUrl` to change only the project base url sounds wrong to me. We can easily add another `projectBaseUrl` convenience method too.